src/ol_concourse/pipelines/libraries/open_api_clients.py

```diff
@@ -147,74 +117,55 @@ def _read_script(script_name: str) -> str:
                 outputs=[Output(name=mit_open_api_clients_repository.name)],
                 run=Command(
                     path="/bin/bash",
-                    args=["-exc", _read_script("open-api-clients-bumpver.sh")],
-                ),
-            ),
-        ),
-        TaskStep(
-            task="git-commit-and-tag",
-            image=git_image.name,
-            config=TaskConfig(
-                platform="linux",
-                inputs=[
-                    Input(name=mit_open_repository.name),
-                    Input(name=mit_open_api_clients_repository.name),
-                ],
-                outputs=[Output(name=mit_open_api_clients_repository.name)],
-                run=Command(
-                    path="/bin/bash",
+                    dir="mit-open-api-clients",
                     args=[
                         "-exc",
-                        _read_script("open-api-clients-tag-release.sh"),
+                        ("scripts/open-api-clients-bumpver.sh"),
```

Comment:
This is expecting the file to exist [here](https://github.com/mitodl/open-api-clients/tree/main/scripts), which it doesn't, it's a file local to this repository, which is why I had written that little `_read_script` function to inline it into command similar to how `bash -exc "$(cat scripts/open-api-clients-bumpver.sh)"` would work.

Reply:
If it's easier to add this script to that repo, feel free to do so